lib/mappers/__tests__/__fixtures__/mapper.figma.js

```diff
@@ -0,0 +1,17 @@
+const sampleOutput = require('../../../../__mocks__/figma/sample_dump.json');
+const { figma2jsonFromJSON, findIds } = require('../../../figma');
+const { uniqueArray } = require('../../../utils');
+
+module.exports = () => {
+  figma2jsonFromJSON(sampleOutput);
+  return ({
+    COLORS : findIds('color'),
```

Comment:
Keeping consistent with the rest of our voice and tone, this sounds more like `findToken` to me.